templates/staffing/prod_report.html

```diff
@@ -42,38 +42,53 @@
 </div>
 
 <div class="row"><br/></div>
-
 <div class="row">
     <div class="col-md-12">
     <div class="table-responsive">
-
     <table class="table table-striped table-hover table-pydici-extra-condensed" id="prod_table">
 
     <tr>
         <th></th>
     	{% for month in months %}
-    	    <th colspan="2" style="border-left:1px solid #C3C3C3;"><center>{{ month|date:"F y" }}</center></th>
+        {% if month.month == today.month and month.year == today.year %}
+        <th colspan="3" style="border-left:1px solid #C3C3C3;"><center>{{ month|date:"F y" }}</center></th>
+        {% else %}
+        <th colspan="2" style="border-left:1px solid #C3C3C3;"><center>{{ month|date:"F y" }}</center></th>
+        {% endif %}
     	{% endfor %}
     </tr>
     <tr>
         <th></th>
         {% for month in months %}
-            <th class="align-right" style="border-left:1px solid #C3C3C3;">{% trans "done" %}</th>
-            <th class="align-right">{% trans "objective" %}</th>
+        <th class="align-right" style="border-left:1px solid #C3C3C3;">{% trans "done" %}</th>
+        <th class="align-right">{% trans "objective" %}</th>
+        {% if month.month == today.month and month.year == today.year %}
+                <th class="align-right">{% trans "end of month" %}</th>
+            {% endif %}
         {% endfor %}
     </tr>
     {% for consultant, warning, charges in data %}
-        <tr class="align-right" {% if not consultant %}style="font-weight: bold"{% endif %}>
-            {% if consultant %}
-                <td style="text-align:left">
-                    {% include "people/__consultant_name.html" %}{% if warning %} <span title="{% trans 'Timesheet does not seem ok' %}" class="pydici-tooltip glyphicon glyphicon-warning-sign"></span>{% endif %}<span title="{{consultant.id}}"></span></td>
-            {% else %}
-                <td style="text-align:left">{% trans "Total" %}</td>
-            {% endif %}
-            {% for status, tooltip, charge in charges %}
-                <td style="background-color:{{ status }}; border-left:1px solid #C3C3C3;"><span class="pydici-tooltip" title="{{ tooltip }}">{{ charge|join:"</span></td><td>" }}</td>
+    <tr class="align-right" {% if not consultant %}style="font-weight: bold"{% endif %}>
+        {% if consultant %}
+        <td style="text-align:left">
+            {% include "people/__consultant_name.html" %}{% if warning %} <span title="{% trans 'Timesheet does not seem ok' %}" class="pydici-tooltip glyphicon glyphicon-warning-sign"></span>{% endif %}<span title="{{consultant.id}}"></span></td>
+        {% else %}
+        <td style="text-align:left">{% trans "Total" %}</td>
+        {% endif %}
+        {% for status, tooltip, charge in charges %}
+            {% for c in charge %}
+                {% if forloop.counter == 1 %} <!-- Only color done charge -->
```

Comment:
This block is quite complex for just adding a column. We may need to rework that a little. Not that simple to discuss in github, maybe tomorow with a coffe cup (if the coffe machine is working again...).